### ISSUE TITLE
Convert `expandable` elements to use HTML Detail tags

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#5]
# Area affected
The entire app. 

# What's the problem?
There is a common pattern for expandable content used throughout the app and the message player pages. It uses a JavaScript-based solution that is not keyboard accessible

# Description of feature
Refactoring the expandable widgets to use the [HTML Details Tag](https://www.w3schools.com/tags/tag_details.asp) would make the elements more accessible for users who are not using a mouse. Using a standard control will also _(in theory)_ make the pages more accessible for people who use screen readers.

![expandable element](https://user-images.githubusercontent.com/342510/101534398-b2888d00-395c-11eb-8e4e-4a997c7e2d6d.gif)

# Alternatives and workarounds
Currently, the expandable elements work with the mouse, which covers the most common use cases.

# Additional context
There are a few places where the expandable widgets are controlled by JavaScript in different ways than tapping/clicking on the label. Care should be taken to make sure there are no regressions.

**Comment by robby1066** [comment]
Basic styling is in place making the `DETAILS` element look pretty much identical to existing expandable elements. Now I just need to go through and swap out all instances.

As in the original description, there are one or two places where the old style expandable elements are preferable, so the code will maintain both options.

**Comment by robby1066** [comment]
With #12 and #28 addressing some other keyboard navigation issues, not being able to access the expandable boxes is starting to stand out like a sore thumb.

**Comment by robby1066** [comment]
Went through and cleaned every instance of this _except_ the message player. That is the one place where the structure is a little weird and may need to be handled differently. There's another ticket for cleaning up the keyboard navigation on that page (which is the main driver for this issue).

I'll test a little more and deploy before closing this.

**Comment by robby1066** [comment]
Deployed on Feb 5, 2021 and verified that all expandable panels are working as expected with both the mouse and the keyboard.
